apps/masterbots.ai/components/routes/profile/user-thread-list.tsx

```
@@ -0,0 +1,62 @@
+    const { data: session } = useSession()
+
+
+    const loadMore = async () => {
+        console.log('🟡 Loading More Content')
+        setLoading(true)
+       let moreThreads: Thread[] = []
+        if(user && !isSameUser(user.userId)){
+          moreThreads = await getBrowseThreads({ userId: user.userId, offset: threads.length });
+        }else{
+         moreThreads = await getThreads({
+          jwt: session!.user?.hasuraJwt,
```

Biome

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)